src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -322,6 +346,24 @@ static bool is_klass_initialized(const TypeInstPtr* vec_klass) {
   return klass->is_initialized();
 }
 
+Node* LibraryCallKit::gen_bitshuffle_operation(int voper, BasicType elem_bt, int num_elem, Node* opd1, Node* opd2) {
```

Comment:
Could you please rename `voper` to `opc` ? The name looks like the opcode expected to be a vector type.

Reply:
ok